hail/python/test/hail/fs/test_worker_driver_fs.py

```diff
@@ -116,3 +116,12 @@ def test_requester_pays_with_project_more_than_one_partition():
     hl.stop()
     hl.init(gcs_requester_pays_configuration='hail-vdc')
     assert hl.import_table('gs://hail-services-requester-pays/zero-to-nine', no_header=True, min_partitions=8).collect() == expected_file_contents
+
+
+@run_if_azure
+@fails_local_backend
+def test_can_access_public_blobs():
+    public_mt = 'hail-az://azureopendatastorage/gnomad/release/3.1/mt/genomes/gnomad.genomes.v3.1.hgdp_1kg_subset.mt'
+    assert hl.hadoop_exists(public_mt)
+    mt = hl.read_matrix_table(public_mt)
+    mt.describe()
```

Comment:
Can you add a test that does `hl.hadoop_open('.../README.txt')` and asserts that the content is what we expect. I'm not so concerned about the content as I am that the `read` path works as expected.